src/main/java/com/study/realworld/article/controller/ArticleController.java

```diff
@@ -21,11 +27,29 @@ public ArticleController(ArticleService articleService) {
         this.articleService = articleService;
     }
 
+    @GetMapping("/articles/{slug}")
+    public ResponseEntity<ArticleResponse> getArticle(@PathVariable String slug) {
+        Article article = articleService.findBySlug(Slug.of(slug));
+        return ResponseEntity.ok().body(ArticleResponse.fromArticle(article));
+    }
+
     @PostMapping("/articles")
     public ResponseEntity<ArticleResponse> createArticle(@RequestBody ArticleCreateRequest request,
         @AuthenticationPrincipal Long loginId) {
         Article article = articleService.createArticle(loginId, request.toArticleContent());
         return ResponseEntity.ok().body(ArticleResponse.fromArticle(article));
     }
 
+    @PutMapping("/articles/{slug}")
+    public ResponseEntity<ArticleResponse> updateArticle(@PathVariable String slug,
+        @RequestBody ArticleUpdateRequest request, @AuthenticationPrincipal Long loginId) {
```

Comment:
인덴트를 동일하게 내리는게 어떨까요
```suggestion
    public ResponseEntity<ArticleResponse> updateArticle(
        @PathVariable String slug,
        @RequestBody ArticleUpdateRequest request, 
        @AuthenticationPrincipal Long loginId) {
```
이게 더 깔끔해보이더라고요!